starter-code/js/index.js

```diff
@@ -1,23 +1,31 @@
-function deleteItem(e){
-
+function getPriceByProduct(itemNode) {
+  var price = document.querySelector(itemNode).innerHTML;
+  return price;
 }
 
-function getPriceByProduct(itemNode){
 
+function createQuantityInput(itemNode) {
+  var qty = document.querySelector(itemNode).value;
+  return qty;
 }
 
-function updatePriceByProduct(productPrice, index){
-
+function getTotalPrice() {
+  var price = getPriceByProduct('.price');
+  var qty = createQuantityInput('.qty');
```

Comment:
A note on this, it will only work for the first product on the list. If you have more products you have to use querySelectorAll